### DISCUSSION TITLE
Time fieldtype format customisation

### DISCUSSION THREAD

**Discussion by byron-roots** [statamic/cms#5600] [Q&A]
Is there anyway to change the time field to show in a 12 hour format, when editing in the cp, instead of the usual 24 hour format?

**Comment by hotmeteor** [comment]
Wondering the same thing...

**Comment by aaronbushnell** [comment]
Submitted the idea here! statamic/ideas#1074

**Comment by jasonvarga** [comment]
This will be possible in 4.53.0
See https://github.com/statamic/cms/pull/9662
